core/src/test/scala/vegas/fixtures/VegasPlots.scala

```diff
@@ -29,7 +29,15 @@ object VegasPlots {
           encodeY2("people", aggregate=Q3)
       )
 
-  val plots: List[SpecBuilder] = ValuePlot :: IQRPlot :: Nil
+  val BinPlot =
+    Vegas("Plot with binning").
+      withDataURL(Movies).
+      mark(Point).
+      encodeY("IMDB_Rating", Quantitative, bin=Bin(maxbins=10.0)).
+      encodeX("Rotten_Tomatoes_Rating", Quantitative, bin=Bin(maxbins = Some(10.0))).
```

Comment:
Unless you were doing this to test it ;) Given it's our own plot, we could test something other than maxbins too?